docs/SupportProgress.md

```diff
@@ -0,0 +1,48 @@
+# The Operators and Functions Support Progress
+Although the Gluten project is still in active development, we still have made big progress on Operators and Functions support.
+- The number of Operators supported is 12, the most common Operators are covered.
+- The number of Functions supported is 94, and Velox supports 127 Functions.
+- The number of Functions commonly used by Spark is 240, and the total number for Spark3.3 is 387.
```

Comment:
How do we measure commonly used function? Is there a common view in the community or the industry? If not, I think we can just record the overall number here. Thanks!

Reply:
@lviiii mentioned there is a page from Spark documents. @lviiii  where is the page?

Reply:
The commonly used functions are referred from [frequently-used built-in functions](https://spark.apache.org/docs/latest/sql-ref-functions.html) , and I add some functions based on my own experience. Maybe, we don't need to mention it. 
The total supported functions' number [for Spark3.3 is 387](https://spark.apache.org/docs/latest/api/sql/) and [for Velox is 204](https://facebookincubator.github.io/velox/functions/coverage.html). Gluten supported 94, in which offloaded 62 is implemented in velox/spark and 32 in velox/presto, shown as below picture.
![image](https://user-images.githubusercontent.com/9353690/207771300-e5efb670-fb1d-4bc2-876b-fb9678cb4497.png)

Reply:
updated